src/Common/src/Interop/Windows/Gdi32/Interop.CombineRgn.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Gdi32
+    {
+        public static class RegionCombineMode
+        {
+            public const int AND = 1;
+        }
+
+        [DllImport(Libraries.Gdi32, ExactSpelling = true, EntryPoint = "CombineRgn")]
+        private static extern int CombineRgnInternal(HandleRef hRgnDest, HandleRef hRgnSrc1, HandleRef hRgnSrc2, int iMode);
+
+        public static int CombineRgn(HandleRef hRgnDest, HandleRef hRgnSrc1, HandleRef hRgnSrc2, int iMode)
```

Comment:
If you can, can you try and change HandleRef parameters to IntPtr wherever we're never putting a wrapper on it?

Reply:
Sadly the only use is
```cs
        public Interop.RegionType CombineRegion(WindowsRegion region1, WindowsRegion region2, Interop.Gdi32.CombineMode mode)
        {
            return Interop.Gdi32.CombineRgn(new HandleRef(this, HRegion), new HandleRef(region1, region1.HRegion), new HandleRef(region2, region2.HRegion), mode);
        }
``